.github/pull-request-template.md

```diff
@@ -0,0 +1,28 @@
+#### Helpful background context
+Describe any additional context beyond what the PR accomplishes if it is likely
+to be useful to a reviewer.
+
+Delete this section if it isn't applicable to the PR.
+
+#### How can a reviewer manually see the effects of these changes?
+Explain how to see the proposed changes in the application if possible.
+
+Delete this section if it isn't applicable to the PR.
+
+#### Includes new or updated dependencies?
+YES | NO
+
+#### What are the relevant tickets?
+Include links to Jira Software and/or Jira Service Management tickets here.
+
+#### Developer
+- [ ] All new ENV is documented in README
```

Comment:
I'm curious what the equivalent is for this project that we have in heroku projects where we have a checkbox to ensure the new settings have been made in heroku... maybe something like "ECS task has been updated with any new ENV"? I'm not sure that's an accurate way to describe it, but essentially a "if we deploy this, is it going to work the first time or are we going to need to futz with ENV for a bit first and if so let's get that out of the way before merge"

Reply:
Hmmm. That's tricky for actual ENV though as that would require terraform updates which are usually a separate ticket for a different team/person.